lib/ui/compositing/scene_host.h

```diff
@@ -11,7 +11,11 @@
 #include "flutter/lib/ui/dart_wrapper.h"
 
 #if defined(OS_FUCHSIA)
+#include "dart-pkg/zircon/sdk_ext/handle.h"
```

Comment:
We should just make this TU and its headers Fuchsia specific. They are not used on any other platform.

Reply:
Done